extensions/sql-database-projects/src/models/project.ts

```diff
@@ -59,6 +60,12 @@ export class Project {
 			const importTarget = this.projFileXmlDoc.documentElement.getElementsByTagName(constants.Import)[i];
 			this.importedTargets.push(importTarget.getAttribute(constants.Project));
 		}
+
+		// find all database references to include
+		for (let r = 0; r < this.projFileXmlDoc.documentElement.getElementsByTagName(constants.ArtifactReference).length; r++) {
+			const uri = this.createProjectEntry(this.projFileXmlDoc.documentElement.getElementsByTagName(constants.ArtifactReference)[r].getAttribute(constants.Include), EntryType.DatabaseReference).fsUri;
+			this.databaseReferences.push(path.parse(uri.fsPath.toString()).name);
```

Comment:
do you need to check for uri being empty/undefined?

Reply:
added validation